pkg/cluster/manifestwork.go

```diff
@@ -482,6 +482,8 @@ func ApplySubManifestWorks(ctx context.Context, kubeClient *kubernetes.Clientset
 		if err != nil {
 			return nil, err
 		}
+
+		return desiredSubscription, nil
```

Comment:
Do you mean that we do not do upgrade automatically?

Reply:
The logic here is to avoid to go to update logic when we just created the object.